infra/README.md

```
@@ -0,0 +1,150 @@
+make test
+
+# Run with hot reload
+make dev
+
+# Stop environment
+make docker-down
+```
+
+### Docker Commands
+
+#### Development Environment
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)